woocommerce/woocommerce
# Checkout block: Add bottom border to expanded order summary on mobile

### Submission Review Guidelines:
### Changes proposed in this Pull Request:

Closes #52807 



### How to test the changes in this Pull Request:

This is a visual change: 

* Test the checkout block on mobile screen size
* Expand the order summary, see that there is now a bottom border separating it from the Contact Information section

Before:

<img width="222" alt="Screenshot 2024-11-20 at 12 19 56 PM" src="https://github.com/user-attachments/assets/f1569d71-3ca2-4adc-acd9-d1952be74abc">


After:

<img width="223" alt="Screenshot 2024-11-20 at 12 18 20 PM" src="https://github.com/user-attachments/assets/b26d2aa2-75ee-43e8-9472-ca1e4b070c52">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Add a bottom border to the checkout block expanded order summary on mobile.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
